include/pushmi/detail/concept_def.h

```diff
@@ -435,14 +459,33 @@ struct And {
     constexpr auto operator!() const noexcept {
         return Not<And>{};
     }
-    template <class That>
-    constexpr auto operator&&(That) const noexcept {
-        return detail::And<And, That>{};
-    }
+    // template <class That>
+    // constexpr auto operator&&(That) const noexcept {
+    //     return detail::And<And, That>{};
+    // }
```

Comment:
In the case where `PUSHMI_AND` expands to `&&`, how will this work without the `operator&&` overloads?

Reply:
because -fconcepts does not use the concepts emulation

Reply:
It does for the things in the `lazy::` namespace that gets used with the `constrain` function.

Reply:
Got it.. yeah, this short-circuits the lazy.. may have been the cause of the transform and tap issues as well.

Reply:
Possibly. When short-circuiting doesn't happen where it's needed, the errors get weird.

Reply:
I tried removing the tap and transform workarounds, but they were still required..